### PR TITLE
Fix multiple API doc links

### PR DESCRIPTION
**Pull request by parlough**
_Description of what this PR is changing or adding, and why:_ Fixes some broken API docs links. A few due to the targets being removed and others due to being rendered enums instead of classes.

_Issues fixed by this PR (if any):_ Fixes N/A

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/release/breaking-changes/eliminating-nullok-parameters.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ return a non-nullable value:
 * [`FocusTraversalOrder.of`][]
 * [`FocusTraversalGroup.of`][]
 * [`Focus.of`][]
-* [`Shortcuts.of`][]
```
Comment (domesticmouse) [inline src/release/breaking-changes/eliminating-nullok-parameters.md:57]: Does `Shortcuts.of` not have a target page like everything else in this list? That seems... unexpected?

**Review by domesticmouse — APPROVED** [review]

File: src/release/breaking-changes/eliminating-nullok-parameters.md (hunk 1 above)
Comment (parlough) [reply src/release/breaking-changes/eliminating-nullok-parameters.md:57]: The `Shortcuts` methods have been removed in a more recent version. Perhaps I should add a small note in parenthesis providing some extra context.

Comment (domesticmouse) [reply src/release/breaking-changes/eliminating-nullok-parameters.md:57]: SGTM. I'm assuming if it is removed in `master` that it was deprecated previously. Maybe Something along the lines of "Deprecated, to be removed in version XXX" or similar.

**Review by sfshaza2 — APPROVED** [review]
lgtm!

**Pull request merged**